guides/v2.1/cloud/reference/docker-config.md

```diff
@@ -0,0 +1,73 @@
+---
+group: cloud
+title: Configure Docker
+version: 2.1
+github_link: cloud/reference/docker-config.md
+functional_areas:
+  - Cloud
+  - Setup
+  - Configuration
+---
+
+{{site.data.var.ece}} provides a Docker environment option for those who use their local environment for develop and testing, or for automation tasks. There are three, essential parts to launching the Magento Cloud Docker environment: a Magento 2 template, Docker Compose, and Magento Cloud Tools.
+
+The [Magento Cloud Docker repository](https://github.com/magento/magento-cloud-docker){:target="\_blank"} contains build information for the following [Docker hub](https://hub.docker.com/r/magento/){:target="\_blank"} images, including the following:
+
+-  PHP: magento/magento-cloud-docker-php
+    -  PHP-CLI - version 7 and later
+    -  PHP-FPM - version 7 and later
+-  NGINX:  magento/magento-cloud-docker-nginx
+
+The Cloud tools provides a `docker:build` command to generate the Docker Compose configuration. Also, you can specify a version using one of the following options:
+
+-  PHP: `--php`
+-  NGINX: `--nginx`
+-  MariaDB: `--db`
+
+#### To launch the Cloud Docker environment:
+
+1.  Download a template from the [Magento Cloud repository](https://github.com/magento/magento-cloud){:target="\_blank"}.
+1.  Add your credentials to `auth.json` file.
+1.  Update the template dependencies.
+
+    ```bash
+    composer install
+    ```
+
+1.  In your local environment, start the Docker configuration generator.
+
+    ```bash
+    vendor/bin/ece-tools docker:build
+    ``` 
+
+1.  Copy the configuration files.
+
+    ```bash
+	cp docker/config.env.dist docker/config.env
+    ```
+
+    ```bash
+	cp docker/global.env.dist docker/global.env
+    ```
+
+1.  Build files to containers and run in the background.
+
+    ```bash
+    docker-compose up -d --build
+    ``` 
+
+1.  Start the Magento installer. This step may take time.
+
+    ```bash
+    docker-compose run cli magento-installer
+    ```
+
+1.  Open the `http://localhost:8080` URL in a browser.
+
+### Integration testing with Cloud tools
+A dedicated Docker environment provides the opportunity for you to customize and exercise the following advantages:
+
```

Comment:
I'm not sure what the intro sentence is saying.

Is it:

Installing Magento Commerce Cloud in a dedicated Docker environment allows you to run integration tests locally to take advantage of the following features and capabilities provided by the Magento Cloud Tools "package"?